lib/moon/assets/age_limit.ex

```diff
@@ -20,7 +20,13 @@ defmodule Moon.Assets.AgeLimit do
   def render(assigns) do
     ~F"""
     <svg
-      class={"moon-age_limit #{@class} #{@click && "cursor-pointer"}"}
+      class={
+        "moon-age_limit",
+        @class,
+        "text-#{@color}": @color,
```

Comment:
The argument itself is classname. So 

`color="text-gohan-100" is passed.`

But, we need your version also (so that legacy code will keep working).

Please also update examples.

Reply:
This version is the one you suggested in my other [PR](https://github.com/coingaming/moon/pull/446#discussion_r1021256035). I think this is for backwards compatibility right?
What do I need to do with the examples? 
The color prop is deprecated so we don't show this usage in the example.